tests/IntegrationTest/InstallUpdateEventsTest.php

```diff
@@ -177,7 +177,7 @@ public function testPluginRunsOnReinstall()
     }
 
     /**
-     * Test that the plugin runs (or doesn't run) when Composer is invoked with the --no-scripts argument.
+     * Test that the plugin runs when Composer is invoked with the --no-scripts argument.
      *
      * Note: the behaviour of Composer changed in 2.1.2. Prior to that, `--no-scripts` would
```

Comment:
I think this "Note" can now be removed as well (as it's no longer relevant).

Reply:
I kept this because it explains why the test exists. Should we instead drop the whole test?

Reply:
Understood and no, let's keep the test as it has value either way.